documentation-website/src/components/card.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {
+  ReactNode,
+} from 'react';
+const HeadingLevels = {
+  H1: 1,
+  H2: 2,
+  H3: 3,
+  H4: 4,
+  H5: 5,
+  H6: 6,
+} as const;
+interface CardProps {
+  titleText: string;
+  titleLevel: (typeof HeadingLevels)[keyof typeof HeadingLevels];
+  sideMode: boolean;
+  children: ReactNode;
```

Comment:
this has to also allow  an Array, otherwise we will never have the case of multiple children